### DISCUSSION TITLE
Relative abundance and Sequence abundance order not equal

### DISCUSSION THREAD

**Discussion by ericvdtoorn** [bluenote-1577/sylph#31] [Q&A] [answered]
Hello Jim, and congratulations on the Nature paper!

We've been using Sylph on some samples, and like the results. One oddity I came across recently is some samples where the the relative abundance was higher for a genus while its sequence abundance was lower. Does this make sense? What should I understand from this?

<div>

<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>clade_name</th>
      <th>relative_abundance</th>
      <th>sequence_abundance</th>
      <th>ANI (if strain-level)</th>
      <th>Coverage (if strain-level)</th>
      <th>sample</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>d__Bacteria|p__Nitrospirota|c__Nitrospiria|o__...</td>
      <td>5.1931</td>
      <td>1.769</td>
      <td>NA</td>
      <td>NA</td>
      <td>AT2</td>
    </tr>
    <tr>
      <th>1</th>
      <td>d__Bacteria|p__Chloroflexota|c__Anaerolineae|o...</td>
      <td>5.1080</td>
      <td>1.840</td>
      <td>NA</td>
      <td>NA</td>
      <td>AT2</td>
    </tr>
  </tbody>
</table>
</div>

**Comment by bluenote-1577** [comment, marked as answer]
@ericvdtoorn sorry for the late response. Please see https://pmc.ncbi.nlm.nih.gov/articles/PMC8184642/  for more information about this. Relative abundance is related to coverage; sequence abundance is related to the # of nucleotides covered. Larger genomes will have "more" sequence abundance relative to smaller genomes.

**Reply by ericvdtoorn** [reply]
I see, thank you!
